app/src/main/java/org/jellyfin/mobile/AppPreferences.kt

```diff
@@ -29,6 +30,14 @@ class AppPreferences(context: Context) {
             }
         }
 
+    var currentUserUuid: UUID?
+        get() = sharedPreferences.getString(Constants.PREF_USER_UUID, null)?.let(UUID::fromString)
+        set(value) {
+            sharedPreferences.edit {
+                if (value != null) putString(Constants.PREF_USER_UUID, value.toString()) else remove(Constants.PREF_USER_UUID)
+            }
+        }
+
```

Comment:
The UUID is actually stored in the database, and I'd personally keep it there, and just cache it in some (injected) manager class. The above `Long` user id is the primary key for the database row.